src/main/java/org/fluentd/logger/sender/RawSocketSender.java

```diff
@@ -179,6 +179,14 @@ private synchronized boolean send(byte[] bytes) {
         return true;
     }
 
+    private void logCannotSendMaxOncePerMinute() {
+        // to not choke on our own log message
+        if(System.currentTimeMillis() + 60 * 1000 > lastCannotSendLogTimestamp) {
```

Comment:
The suppression time 60 seconds seems too long to me. Could you make this value configurable (default: 10 seconds)?